amber-slsa-buildtype.json

```diff
@@ -0,0 +1,137 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Amber SLSA buildType",
+  "description": "Describes a SLSA buildType, as defined in https://slsa.dev/provenance/v0.2.",
+  "type": "object",
+  "required": [
+    "buildType",
+    "buildConfig",
+    "materials"
+  ],
+  "not": {
+    "required": [
+      "invocation"
+    ]
+  },
+  "properties": {
+    "buildType": {
+      "const": "https://github.com/project-oak/transparent-release/amber-slsa-buildtype.json"
```

Comment:
I think it is important to have versioning here, so having a directory structure like the one I suggested is required.